modules/core/arrow-core-data/src/test/kotlin/arrow/core/extensions/NumberSemiringTest.kt

```diff
@@ -1,80 +1,31 @@
 package arrow.core.extensions
 
 import arrow.test.UnitSpec
+import arrow.test.generators.byte
+import arrow.test.generators.short
 import arrow.test.laws.SemiringLaws
+import io.kotlintest.properties.Gen
 import io.kotlintest.shouldBe
 
 class NumberSemiringTest : UnitSpec() {
```

Comment:
Is this needed? isn't it covered by
>   SemiringLaws.laws(
        Schedule.semiring<ForId, Any?, Int>(Int.monoid(), Id.monad()),
        Schedule.genK<ForId, Int>(Id.monad()),
        eqK(Id.eqK(), Id.monad(), 0 as Any?).liftEq(Int.eq())
      ),

Reply:
Not exactly, it's covering a Schedule with Int specification, but not the laws for each number type as here